lib/pipelines.js

```diff
@@ -476,6 +476,32 @@ async function http_create_pipeline_promotion(pg_pool, req, res, regex) {
     create_pipeline_promotion_target_query(pg_pool, [pipeline_promotion_target_id, created, created, pipeline_promotion_uuid, release.id, app_dst.app]).catch((err) => {
       console.warn('An error occured trying to create a pipeline promotion target:', err)
     });
+    // Send out a 'pipeline_promotion' hook on the current target app
+    setTimeout(() => common.notify_hooks(pg_pool, target.app_uuid, 'pipeline_promotion', JSON.stringify({
+      'action': 'pipeline_promotion',
+      'app': {
+        'name': target.app_name,
+        'id': target.app_uuid,
+      },
+      'space': {
+        'name': target.space_name,
+      },
+      'promoted_from': {
+        'app': {
+          'name:': app.app_name,
+          'id': app.app_uuid,
+        },
+        'space': {
+          'name': app.space_name,
+        },
+      },
+      'release': {
+        'id': release.release_id,
+        'build': release.build_uuid,
```

Comment:
The documentation does not mention that the build uuid is included, in addition, if we keep the build id can we pull it out from the release object and have `"build":{"id":release.build_uuid}` off of the root? Reason why, is if we decide to add more information about the build (such as the sha, repo, or commit message or something else) we don't have to modify or remove properties from the structure of the payload. Makes it easier to add new information to payloads without breaking everyone.

Reply:
Ah yeah I gotcha. Will do